unittests/test_meta.py

```diff
@@ -3,23 +3,37 @@
 #
 # SPDX-License-Identifier: BSD-3-Clause
 
-import reframe as rfm
+import pytest
+
 import reframe.core.meta as meta
+import reframe.core.deferrable as deferrable
+
+from reframe.core.exceptions import ReframeSyntaxError
+
+
+@pytest.fixture
+def MyMeta():
+    class Foo(metaclass=meta.RegressionTestMeta):
+        pass
+
+    yield Foo
```

Comment:
Why is extra metaclass needed?

Reply:
It's not needed, but I thought it was more convenient than writing `Foo(metaclass=meta.RegressionTestMeta)` every time.

Reply:
Indeed, could you add a small one line comment saying that this is just for convenience?